### DISCUSSION TITLE
Publishing open-source libs to APM

### DISCUSSION THREAD

**Discussion by j3k0** [airsdk/apm#25] [General]
I expect a few open-source actionscript libs have authors that might not publish their packages to APM immediately. Then someone will probably do that on their behalf.

In the `package.json` file, it might be useful to identify the library author(s), as opposed to the package maintainer(s). This way it will be possible to correctly attribute a lib to its authors in the repository, even when published by another person/entity.

I guess as a user, you mostly need to trust the publisher of the lib though.

Any thoughts?

**Comment by marchbold** [comment]
There is currently no "author" identification in the package.json, only details on the package (docs/url etc). This should always relate to the package and not the author or publisher.  

We may add something in the future but there's nothing in there now.  The only information is on the site, that being the publisher, i.e. the person who pushed the package to the site.

**Reply by j3k0** [reply]
OK, so is there any issues (ethical, political) with me publishing open-source libs like Feathers, Citrus Engine, if they are not available on `apm`?

Can control can be transferred to the original authors if they decide later on to take control of those packages?

**Reply by marchbold** [reply]
I would prefer you only publish packages you author / control.

We are planning on setting up a "generic" account to publish general packages that are requested from the community where the authors aren't available so if you want anything feel free to prepare a package and send it to us. 

Also Josh is already preparing a feathers package.
